homework/pages/CalcForm.js

```diff
@@ -0,0 +1,20 @@
+class CalcForm {
+
+    constructor(page) {
+      this.page = page;
+    }
+
+    async enterFirstNumber(number) {
+        await this.page.locator('#number1Field').type(number);
```

Comment:
here you are using type() function to enter the value you need. Why use this instead of fill () function that has more checks? Now even when the field is not visible, the tests think they can fill the value and you have added no additional checks that that is possible

Reply:
Thanks for comment